hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -1576,14 +1582,6 @@ public String toString() {
         return "IMap{" + "name='" + name + '\'' + '}';
     }
 
-    @Override
-    protected void onInitialize() {
-        super.onInitialize();
```

Comment:
no delete only moved it up https://github.com/hazelcast/hazelcast/pull/9646/files/21d7af8bc84afa61c7e99b6f967ceb7f62b4aad8#diff-3540ff2664ebe82892440b80a7293680R250